bin/fmfexporter

```diff
@@ -54,4 +56,12 @@ if __name__ == '__main__':
         sys.exit(0)
 
     # Submitting filtered test cases
-    adapter.submit_testcase_list(tc_list)
+    submitted_tc_list = adapter.submit_testcase_list(tc_list, parsed.jira_populate_tc)
```

Comment:
@fgiorgetti this is what you did not like last time. As said, parsed value is passed there by default ;-)

Reply:
I dont think we should add references to the Jira stuff in the fmfexporter bin.
It should be strictly contained inside the polarion adapter, meaning everything
under fmfexporter/adapters/polarion (for example the connectors package).

Same for the submit_testcase_list method. It should not have any reference to
the Jira connector here. 

FMF stuff must be generic and only the adapter will have references to the other
tools.

Reply:
I will try to figure out a way, how to do it more generic.. but imo, submitting defects to jira/polarion is imo general stuff.